src/ApiService/ApiService/OneFuzzTypes/Model.cs

```diff
@@ -917,7 +917,7 @@ public record VmDefinition(
     long Value
 );
 
-public record TaskDefinition(
+public readonly record struct TaskDefinition(
```

Comment:
General question and not a comment: What's the benefit of changing this to a struct?

From what I read [here](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/choosing-between-class-and-struct), structs should be smaller than 16 bytes.

Reply:
It was so I can make it `readonly`, but `record` (classes) are also readonly for positional parameters, so I can undo this. 🙂